src/test/java/uk/ac/cam/cl/dtg/segue/etl/ContentIndexerTest.java

```diff
@@ -77,8 +76,6 @@ public final void setUp() throws Exception {
   /**
    * Test that the buildSearchIndex sends all of the various different segue data
    * to the searchProvider and we haven't forgotten anything.
-   *
-   * @throws Exception
```

Comment:
Wasn't this correct? It still looks like it's throwing an exception.

Reply:
Hmmm... that one probably should actually. It complains if there's no description on the tag but I suppose I can add a generic summary...
Actually should the methods specify which exceptions could occur or do we not particularly care about that for test methods if that's not the subject of the test case?

Reply:
I normally wouldn't even bother with Javadoc in the tests, I prefer to make sure that the method name is descriptive enough.